src/eachMinuteOfInterval/index.ts

```diff
@@ -0,0 +1,75 @@
+import addMinutes from '../addMinutes/index'
+import toDate from '../toDate/index'
+import startOfMinute from '../startOfMinute/index'
+import requiredArgs from '../_lib/requiredArgs/index'
+
+interface Interval {
+  start: Date | number;
+  end: Date | number;
+}
+
+interface Options {
+  step?: number
+}
+
+/**
+ * @name eachMinuteOfInterval
+ * @category Interval Helpers
+ * @summary Return the array of hours within the specified time interval.
+ *
+ * @description
+ * Return the array of minutes within the specified time interval.
+ *
+ * @param {Interval} interval - the interval. See [Interval]{@link docs/types/Interval}
+ * @param {Object} [options] - an object with options.
+ * @param {Number} [options.step=1] - the step to increment by. Th starts of hours from the hour of the interval start to the hour of the interval end
```

Comment:
I think it's a typo here
```suggestion
 * @param {Number} [options.step=1] - the step to increment by. The starts of hours from the hour of the interval start to the hour of the interval end
```